src/boot/efi/boot.c

```diff
@@ -911,6 +911,27 @@ static VOID config_entry_free(ConfigEntry *entry) {
         FreePool(entry);
 }
 
+static VOID config_remove_by_type(Config *config, enum loader_type type) {
```

Comment:
i appreciate that you split this out in a separate commit, but this way the code won't be bisectable anymore, since gcc will notice that the function is unused and we told it to make that a compile error.

so the function must be added and somehow be referenced for things to remain fully bisectable

Reply:
Must be for userspace only. I only get a warning for unused functions.

Reply:
In fact, I see no `-Werror=unused-function` in userspace meson.build either.